### PR TITLE
Logs for large messages + metrics on size.

### PR DESCRIPTION
**Pull request by eskimor**

### CODE REVIEW

File: node/network/statement-distribution/src/lib.rs [hunk 1]
```diff
@@ -949,23 +964,24 @@ fn statement_message(
 }
 
 /// Check whether a statement should be treated as large statement.
-fn is_statement_large(statement: &SignedFullStatement) -> bool {
+///
+/// Also report size of statement - if it is a `Seconded` statement, otherwise `None`.
+fn is_statement_large(statement: &SignedFullStatement) -> (bool, Option<usize>) {
```
Comment (drahnr) [inline node/network/statement-distribution/src/lib.rs:969]: nit: using a tiny helper `struct IsLarge {}` with `Deref<Target=bool>` would improve legibilty _possibly_

**Review by sandreim — APPROVED** [review]

**Comment by eskimor** [comment]
bot merge

**Comment by drahnr** [comment]
github is down~ish again

**Comment by eskimor** [comment]
bot force merge

**Comment by sandreim** [comment]
bot merge

**Pull request merged**